modules/kernel/src/main/scala-2.10/notebook/ReplCalculator.scala

```diff
@@ -361,7 +361,22 @@ class ReplCalculator(
       val result = scala.concurrent.Future {
         // this future is required to allow InterruptRequest messages to be received and process
         // so that spark jobs can be killed and the hand given back to the user to refine their tasks
-        val result = repl.evaluate(newCode, msg => thisSender ! StreamResponse(msg, "stdout"))
+        val cellId = er.cellId
+        def replEvaluate(code:String, cellId:String) = {
+          val cellResult = try {
+           repl.evaluate(s"""
+              |sparkContext.setJobGroup("cell-$cellId", "Created by Spark Notebook")
```

Comment:
I'd replace the "created by Spark Notebook" with a cell code snippet (need to check the length limit), so `Description` in `Spark UI` would be even more valuable!

<img width="1248" alt="screen shot 2015-12-20 at 23 37 49" src="https://cloud.githubusercontent.com/assets/213426/11920276/fda3c998-a772-11e5-81b8-e13567ca99d2.png">